ftplugin/markdown.vim

```diff
@@ -0,0 +1,32 @@
+
+
+" plasticboy/vim-markdown {{{1
+
+hi! link htmlH1 DraculaPurpleBold
+
+hi! link htmlItalic DraculaYellowItalic
+hi! link mkdItalic DraculaYellowItalic
+
+hi! link htmlBold DraculaOrangeBold
+hi! link mkdBold DraculaOrangeBold
+
+hi! link htmlBoldItalic DraculaOrangeBoldItalic
+hi! link mkdBoldItalic DraculaOrangeBoldItalic
+
+hi! link mkdRule DraculaComment
+
+hi! link mkdBlockquote DraculaYellowItalic
```

Comment:
In standard dracula, this is `DraculaCyan`

Reply:
Nope. Good catch!

Reply:
Yeah, so both are technically incorrect as it should be cyan for the `>` and yellow italic for the actual text.

The built-in syntax only targets the `>` so that's why it's cyan.

This one only targets everything, with no distinction of the `>`, so I opted to make that yellow italic. Thoughts?